sysbuild/Kconfig.flprcore

```diff
@@ -4,7 +4,7 @@
 
 config SUPPORT_FLPRCORE
 	bool
-	default y if (SOC_NRF54L15_CPUAPP || SOC_NRF54H20_CPUAPP)
+	default y if (SOC_NRF54L15_CPUAPP || SOC_NRF54H20_CPUAPP || SOC_NRF54LM20A_ENGA_CPUAPP)
```

Comment:
You can consider adding this small change in a dedicated PR as it  causes the whole CI to run